docs/guides/prefect-deploy.md

```diff
@@ -466,7 +466,7 @@ Use the tabs below to explore these two deployment creation options.
 
     !!! tip "`prefect.yaml` file flexibility"
         In older versions of Prefect, this file had to be in the root of your repository or project directory and named `prefect.yaml`.
-        Now this file can be located elsewhere and be named differently. To use a custom `prefect.yaml` file, supply the new 
+        Now this file can be located elsewhere and be named differently. You can even have multiple `prefect.yaml` files with the same name in different directories. To use a custom `prefect.yaml` file, supply the new 
```

Comment:
by "elsewhere" it still needs to be in the root dir right? just don't want anyone thinking you could go from like a link to another repo

Reply:
It can be outside of the root dir (i.e in a parent dir  `../prefect.yaml` or subdir `./dev/prefect.yaml`. But it does need to be in a directory. What do you think of this change:

```diff
-- Now this file can be located elsewhere and be named differently.
++ Now this file can be located in another directory and be named differently.
```

Reply:
i didn't know it could be outside the root dir, can we just say that explictly? 👼

Reply:
Done!

Reply:
ty!